specifyweb/frontend/js_src/lib/components/Forms/DeleteButton.tsx

```diff
@@ -77,16 +79,27 @@ export function DeleteButton<SCHEMA extends AnySchema>({
 
   return (
     <>
-      <ButtonComponent
-        title={isBlocked ? formsText.deleteBlocked() : undefined}
-        onClick={(): void => {
-          handleOpen();
-          setDeferred(false);
-        }}
-      >
-        {isBlocked ? icons.exclamation : undefined}
-        {commonText.delete()}
-      </ButtonComponent>
+      {isIcon ? (
+        <Button.Icon
+          title={isBlocked ? formsText.deleteBlocked() : commonText.delete()}
```

Comment:
add aria-label too (whenever interactive element does not have a text content or an associated label (in case of input field), it must have aria-label)
in this case, the aria-label can be the same as the title

Reply:
ah forgot that I made it automatic. see this:
https://github.com/specify/specify7/blob/b5d64d14e659c3e8c50d0200f80954725cc114f4/specifyweb/frontend/js_src/lib/components/Atoms/Button.tsx#L134